### DISCUSSION TITLE
Wyze Cam Pan v2 RTSP Errors

### DISCUSSION THREAD

**Discussion by Psyringe** [gtxaspec/wz_mini_hacks#317] [Q&A] [answered]
Can't get RTSP to function on the Wyze Cam Pan v2.

Standard installation with no configuration the device boots. If no changes are made to the files on the micro sd card the device continues to boot.

If the micro sd card is removed and the wz_mini.conf file is altered, specifically the RTSP settings, the device remains locked on the purple light ( red + blue ) during boot.

If the wz_mini.conf file is altered on a fresh install, specifically the RTSP settings, the device remains locked on the purple light ( red & blue ) during boot with no install dialog.

Studied the instructions and am unable to find an error in method.

The device is currently on the latest firmware, confirmed to function in thread: https://github.com/gtxaspec/wz_mini_hacks/discussions/155

The device was tested with earlier firmware to the same results.

Can only think to attempt changing the wz_mini.conf file remotely after a fresh unmodified install, however no idea why this would be different than previous methods.

Any insight to getting my setup functioning?

**Comment by gtxaspec** [comment]
what program are you using to edit the file? make sure you use a program that is compatible with unix line endings.   windows notepad won't work.  I would recommend using the ssh terminal and using nano or vi to edit the conf file.

**Reply by Bob-AOL** [reply]
Don't know why notepad has become a big no-no recently. It has worked fine for me.. 

The other option would be to try using the web interface cgi script by enabling the webserver with notepad prior to booting from the sd card. If that works, the .conf file can then be further configured via the web interface if ssh or nano are not part of your immediate skill set. Just don't try to reboot from the web interface as it _definitely_ has a history of issues. Use the app instead.

**Reply by Psyringe** [reply]
This was the issue, all changes made using Microsoft WordPad.

Solution varied and might be more user friendly.

**Reply by Bob-AOL** [reply]
WordPad and plain old Notepad are entirely different animals. I've never had any issue using basic Notepad for editing wz_mini.conf.

**Reply by Psyringe** [reply]
Admittedly was in a hurry and overlooked formatting, only working in this type of environment a handful times a year.

Notepad was my first choice but fed a single line of text (Windows 7), not conducive to my state of mind at the time.

**Comment by Psyringe** [comment, marked as answer]
Used notepad2-mod to save the document with  Unlix (LF) Line Endings.

Speculate editors such as Notepad++ or Visual Studio can resolve this issue for unsuspecting users in the Windows environment accessing the micro sd card directly on the computer.

**Reply by Bob-AOL** [reply]
Again, plain old Notepad works fine for me editing the .conf file on a laptop or directly on the SD card.
